Sources/Tabman/Bar/BarButton/Types/TMLabelBarButton.swift

```diff
@@ -156,6 +156,7 @@ open class TMLabelBarButton: TMBarButton {
     }
     
     open override func update(for selectionState: TMBarButton.SelectionState) {
+        super.update(for: selectionState)
```

Comment:
I believe this missing call to super was a bug anyway? Or was it intentional? Anyway, it is required to correctly update the `selected` accessibility trait for the button.

Reply:
@iv-mexx it was actually intentional (didn't want the default transition) - but it's not so nice and also needed now. I'll refactor the default transition into something a bit nicer than avoiding a `super` call 👍

Reply:
Alright 👍